Nos/Views/Components/WizardNavigationStack.swift

```diff
@@ -16,7 +16,7 @@ struct WizardNavigationStack<Root>: View where Root: View {
     var body: some View {
         NavigationStack(root: root)
             .frame(idealWidth: 320, idealHeight: 480)
-            .presentationDetents([.medium])
+            .presentationDetents([.height(480)])
```

Comment:
For some reason I feel more comfortable using fractions than specific heights -- I guess because we can say we want something like 60% or 75% or 80% of the screen height, depending on what looks and works best. Could we consider using the fraction detent? https://developer.apple.com/documentation/swiftui/presentationdetent/fraction(_:)

Reply:
Let me think about it. The content of the sheet is more or less fixed: unless the user changes some global accessibility settings like text size, the height will vary according the available device width. Bigger screens have more height and width, and thus, are capable of displaying longer texts in one line. This means that bigger screens will be able to display the sheet with less height, not more. So, using fractions, in this specific case, will undermine the advantage of having a big screen, and display the sheet with more height than needed.

I don't like the fixed height as well. To be honest, the real solution would be to rework the design so that the content can be scrolled up and down while the done button is always visible (like it would if we display it in the navigation bar instead of the bottom of the sheet). In this way, we would even accommodate to users who chose larger text sizes in iOS. So, if we want to really fix this, I think the next step would be to convince others to work on that. Up until then, we should be fine with this ugly workaround (which was not even my first choice either).

Reply:
Yeah, I guess if a height of 480 is needed for the smallest screen, it should be totally fine on the biggest screen. I think I'm good with keeping this as-is; I just wanted to think through it.

I just assumed that the design should be a percentage of the screen, but I think that doesn't actually work for this case.